docs: Update nrfcloud multi service sample readme (#20429)
nrfconnect/sdk-nrf: samples/cellular/nrf_cloud_multi_service/README.rst

1. Make sure your device is plugged in and that this sample has been flashed to it.
#. Install the device and server credentials using the :file:`device_credentials_installer.py` Python script :ref:`you installed <nrf_cloud_multi_service_install_nrf_utils>`:

   .. note::
      The :file:`device_credentials_installer.py` Python script requires other scripts in the same repository to be present in the same folder on your computer.

>       Please clone this repository in its entirety.
>       The script above will not work by itself without the others.
>       Also, please install the required Python packages as mentioned in the script documentation.

Reviewer: ```suggestion
      Clone this repository in its entirety.
      The script does not work by itself without the others.
      Also, install the required Python packages as mentioned in the script documentation.
```
